frontend/class-opengraph-image.php

```diff
@@ -243,8 +273,6 @@ private function set_singular_image( $post_id = null ) {
 		if ( $this->has_images() ) {
```

Comment:
I think these lines have now become abundant.

Reply:
These are necessary for early returns. Due to the current structure op this class, there is no other way of exiting out of this.